src/types.js

```diff
@@ -29,7 +29,7 @@ export type VersionType = {
 export type EntryKindType = 'addition' | 'change' | 'fix';
 
 export type EntryType = {
-  component: string,
+  component: string | null,
```

Comment:
...was what I did before but this includes `undefined` (which I explicitly did not want to allow)